src/cli.ts

```diff
@@ -173,6 +176,22 @@ function makeCommand (process: NodeJS.Process): Command {
     ).default(
       Enums.ComponentType.Application
     )
+  ).addOption(
+    new Option(
+      '-w, --workspace <workspace...>',
+      'Whether to only include dependencies for a specific workspace. ' +
+      '(can be set multiple times)'
+    ).default([], 'empty')
+  ).addOption(
+    new Option(
+      '-no-ws, --no-workspaces',
+      'Do not include dependencies for workspaces.'
+    )
+  ).addOption(
+    new Option(
+      '-wr, --include-workspace-root',
```

Comment:
where does the `-wr`  come from? 

it is intended to use similar args like `nom ls` does, so adoption is easy.
which version of npm knows `-wr` here?